openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.component.tsx

```diff
@@ -63,9 +64,9 @@ const Explore: React.FC<ExploreProps> = ({
 
   // get entity active tab by URL params
   const defaultActiveTab = useMemo(() => {
-    const entityName = ENTITY_PATH[tab as keyof typeof ENTITY_PATH] ?? 'table';
+    const entityName = toUpper(ENTITY_PATH[tab as EntityType] ?? 'table');
```

Comment:
What's the need of doing `toUpper` here?

Reply:
@chirag-madlani The object which is using `entityName`  is expecting entity name in Uppercase to return it's value